site/src/pages/DeploymentSettingsPage/OAuth2AppsSettingsPage/OAuth2AppForm.tsx

```diff
@@ -54,17 +88,54 @@ export const OAuth2AppForm: FC<OAuth2AppFormProps> = ({
 					autoFocus
 					fullWidth
 				/>
-				<TextField
-					name="callback_url"
-					label="Callback URL"
-					defaultValue={app?.callback_url}
-					error={Boolean(apiValidationErrors?.callback_url)}
-					helperText={
-						apiValidationErrors?.callback_url ||
-						"The full URL to redirect to after a user authorizes an installation."
-					}
-					fullWidth
-				/>
+
+				{/* Redirect URIs Section */}
+				<Stack spacing={1}>
+					<div css={{ display: "flex", alignItems: "center", gap: 8 }}>
+						<span css={{ fontWeight: 500, fontSize: 14 }}>Redirect URIs</span>
+						<IconButton
+							size="small"
+							onClick={addRedirectUri}
+							css={{ padding: 4 }}
```

Comment:
```suggestion
							className="p-1"
```

same here, and for reference one tailwind "spacing" is basically 4px